### DISCUSSION TITLE
Mapping Zones to Counties

### DISCUSSION THREAD

**Discussion by su-rowen** [weather-gov/api#257] [Q&A]
**Please describe what you're trying to do.**
I am trying to map forecast zones to their respective counties.

I found these files which are exactly what I need. https://www.weather.gov/gis/ZoneCounty

Is there a way to get this data from the api.weather.gov API?

**Comment by formula4** [comment]
I have all of the current Forecast, Fire, and County zones as simplified geoJSON (Slightly less accurate but way better for bandwidth use). This took me quite some time to figure out but I have nailed the process at last. Would these help you?

**Comment by JJKraw** [comment]
I don't know of an API mapping specifically of zones to/from counties, but for any given point you can find out both.

Also, in case you aren't aware, there are a few cases of forecast zones belonging to multiple counties. Ones I know of:

- MDZ011 (Southern Baltimore) contains part of Baltimore county (MDZ005) and all of Baltimore City County (MDZ510).
- All of Puerto Rico.
